File: A. Next Round/A. Next Round.py
participants_and_minimum_score = list(map(int, input().split()))

participants = list(map(int, input().split()))

number_of_participants = participants_and_minimum_score[0]
minimum_score = participants_and_minimum_score[1]

score_limit_position = participants[minimum_score - 1]

advancers = 0

for participant in participants:
    if participant > 0 and participant >= score_limit_position:
        advancers += 1

print(advancers)

File: A. Next Round/A. Next Round.cpp
#include <iostream>
#include <vector>

using namespace std;


int main() {
    int number_of_participants, minimum_score;

    cin >> number_of_participants >> minimum_score;


    vector<int> participants(number_of_participants);
    for (int i = 0; i < number_of_participants; i++) {
        cin >> participants[i];
    } 

    int score_limit_position = participants[minimum_score - 1];
    int advancers = 0;

    for (int i = 0; i < number_of_participants; i++) {
        if (participants[i] > 0 && participants[i] >= score_limit_position) {
            advancers++;
        }
    }

    cout << advancers << endl;

    return 0;

}